packages/mermaid/src/docs/community/docker-development.md

```diff
@@ -0,0 +1,109 @@
+---
+prev:
+  text: 'FAQ'
+  link: 'config/faq'
+next:
+  text: 'Contributing Code'
+  link: '/community/code'
```

Comment:
Would it be easier to just have the Docker instructions in the same file as the non-docker instructions?

E.g. something like:

```markdown
## Get the Source Code

## Technical Requirements

> The following documentation describes how to work with Mermaid in your host
> environment. There's also a
> [Docker installation guide](#docker-development) if you prefer to
> work in a Docker environment.

## Setup and Launch

### Switch to project

### Install packages

### Launch

## Verify Everything is Working

## Docker Development

> The following documentation describes how to work with Mermaid in a Docker
> environment. Go back to
> [Technical Requirements](#technical-requirements) if you want to
> to work in your host environment, without Docker.

### Docker Technical Requirements

### Make `./run` executable

### Install packages in Docker

### Launch in Docker

### Verify Everything is Working in Docker
```

Reply:
I don't know. In terms of supporting this - many be. But I think combining would rather distract readers. There are many similar parts inside, some phrases are repeated too. I'd rather stick to separate page instead of a long and bewildering one. I don't actually know. I am not a technical writer.

Reply:
> I think I am the only one who develops it in Docker...

I'm generally using Docker too (well, `podman`, but from a user's perspective, Podman is pretty similar to Docker).

Although I'm currently running all the commands manually, which works pretty well, except for opening the Cypress GUI. It looks like the `./run` script handles Cypress pretty well, so the next time I need to debug a Cypress E2E test, I'll probably read your docs on how to get it working in Docker :smile:.